dydx3/constants.py

```diff
@@ -68,7 +68,7 @@
 # ------------ Asset IDs ------------
 ASSET_ID_MAP = {
```

Comment:
outside the scope of this PR, but will eventually have to be based on `network_id`